designs/administrator/olm/sort-by-recently-updated/sort-by-recently-updated.md

```diff
@@ -0,0 +1,48 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Sort by Provider update on OperatorHub
+
+When viewing the list of Operators available for install on OperatorHub, the users are interested in which Operators were updated most recently. OperatorHub should support sorting by the most recently updated Operator.
+
+## Sorting on OperatorHub
+
+The Sort by menu and View icons allow the user to now further customize their sorting experience on Operator Hub.
+
+![Sort-by-on-OperatorHub](img/1-filter-on-operator-hub.jpg)
+
+## User selects “Provider update” on menu
+
+When the user selects “Provider update” from the Sort by menu, the information about when this operator was last updated appears in the bottom left of the card.
+
+![Sort-by-on-OperatorHub2](img/2-filter-on-operator-hub.png)
+
+## “Updated by provider” appears on card under Operator Title and Provider
+
+Now the Sort by dropdown has been updated with the Provider update selection, and the timestamp appears under Operator Title and Provider.
```

Comment:
It makes me wonder if this is important information to always show. I guess PM would have a better sense of that than me though :)

Reply:
Because we have so much competing information for our Operator Hub tiles it seems that ux and PM alike have aligned on the idea that we will only show relevant information when the user signals they want it. I see your below comment and will answer that one, too.

Reply:
Thanks!!